src/System.Configuration.ConfigurationManager/tests/System/Configuration/SettingElementTests.cs

```diff
@@ -46,11 +46,11 @@ public void DefaultValueIsNull()
         }
     
         [Fact]
-        public void DefaultSettingElementExceptionForGetHashCode()
+        public void DefaultConstructorEquality()
         {
-            var Element = new SettingElement();
-            Assert.Throws<NullReferenceException>(() => Element.GetHashCode());
-            //likely a bug
+            var ElementOne = new SettingElement();
+            var ElementTwo = new SettingElement();
+            Assert.True(ElementOne.Equals(ElementTwo));
```

Comment:
This is a good test, but it's not testing your change. A test for your change would be `Assert.Equals(ElementOne.GetHashCode(), ElementTwo.GetHashCode());`